app/verify-email/page.tsx

```diff
@@ -1,67 +1,116 @@
-import React from 'react'
-import Link from 'next/link'
-import { ArrowRight } from 'lucide-react'
+'use client'
+
+import React, { useEffect, useState } from 'react'
+import { ArrowRight, CheckCircleIcon, Frown, Loader2, Mail } from 'lucide-react'
+
+function VerifyEmail() {
+  const [isSubmitting, setIsSubmitting] = useState(false)
+  const [submitSuccess, setSubmitSuccess] = useState(false)
+  const [submitFailed, setSubmitFailed] = useState(false)
+
+  const [workEmail, setWorkEmail] = useState<string | null>(null)
+
+  useEffect(() => {
+    try {
+      localStorage.getItem('workEmail')
+      if (workEmail) {
+        setWorkEmail(workEmail)
+      } else {
+        setSubmitFailed(true)
+      }
+    } catch (error) {
+      console.error('Error fetching work email from local storage', error)
+      setWorkEmail(null)
+      setSubmitFailed(true)
+    }
+  }, [])
+
+  const handleResendVerificationEmail = async () => {
+    setIsSubmitting(true)
+    setSubmitFailed(false)
+
+    const payload = {
+      email: workEmail,
+    }
+
+    try {
+      const response = await fetch('https://api.signoz.cloud/v2/users/notify', {
+        method: 'PUT',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(payload),
+      })
+
+      if (response.ok) {
+        setSubmitSuccess(true)
+
+        setTimeout(() => {
+          setSubmitSuccess(false)
```

Comment:
why this 5 seconds timeout ?

Reply:
Showing the message for 5 seconds and then enabling the button after 5 secs